.github/workflows/test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.8', '3.9', '3.10']
-        os: ['ubuntu-20.04', 'macos-11', 'windows-2019']
+        os: ['macos-11']
```

Comment:
Why remove these?

Reply:
Because I used locally compiled wheels for MacOS. This is temporary until the non-void return types and classical ops are released in `pyqir`. Currently, this is merged in the `main` branch only.